coalib/misc/Exceptions.py

```diff
@@ -15,6 +16,9 @@ def get_exitcode(exception, log_printer=None):
         exitcode = 0
     elif isinstance(exception, SystemExit):
         exitcode = exception.code
+    elif isinstance(exception, ContextualVersionConflict):
+        print("Version Conflict Error. Check 'pkg_resources'.")
```

Comment:
I would change the message, this is not really helpful. If one works in a big virtualenv, they cannot recheck all the packages. I think it would be best to either output the whole error message or at least extrapolate the actual pacakge name and version that need to be fixed.